ODROID-C2.md

```diff
@@ -0,0 +1,14 @@
+## Additional Packages
+
+* X.org video driver package: xf86-video-fbdev
+* Mali EGL and GLES drivers (currently framebuffer only): odroid-c2-libgl-fb
+* wiringPi modified for the ODROID-C1/C2: wiringc1
+
+## Display
+
+Configuring the resolution of the screen on the HDMI port is done through two items
+
+* /boot/boot.ini
+* /usr/bin/amlogic.sh (called by amlogic.service)
```

Comment:
Perhaps _how_ I am describing it is the issue. The settings are currently done only though boot.ini. amlogic.sh could be altered in the future to allow those values to be taken from something other than thant ``/proc/cmdline``, allowing it to be more directly used. Sure, ideally this would be replaced entirely and configured from a kernel module or EDID, KMS, etc.